kview/src/main/java/dev/ikm/komet/kview/klfields/floatfield/DefaultKlFloatField.java

```diff
@@ -0,0 +1,30 @@
+package dev.ikm.komet.kview.klfields.floatfield;
+
+import dev.ikm.komet.framework.observable.ObservableField;
+import dev.ikm.komet.framework.view.ObservableView;
+import dev.ikm.komet.kview.controls.KLFloatControl;
+import dev.ikm.komet.kview.controls.KLReadOnlyStringControl;
+import dev.ikm.komet.kview.klfields.BaseDefaultKlField;
+import dev.ikm.komet.layout.component.version.field.KlFloatField;
+import javafx.scene.Node;
+
+public class DefaultKlFloatField extends BaseDefaultKlField<Float> implements KlFloatField {
+
+    public DefaultKlFloatField(ObservableField<Float> observableFloatField, ObservableView observableView, boolean isEditable) {
+        super(observableFloatField, observableView, isEditable);
+
+        Node node;
+        if (isEditable) {
+            KLFloatControl floatControl = new KLFloatControl();
+            floatControl.valueProperty().bindBidirectional(observableFloatField.valueProperty());
+            floatControl.setTitle(getTitle());
+            node = floatControl;
+        } else {
+            KLReadOnlyStringControl readOnlyStringControl = new KLReadOnlyStringControl();
+            readOnlyStringControl.setText(String.valueOf(observableFloatField.value()));
```

Comment:
Can we create a new Bindings to the textProperty here?
Here is an example on stack overflow.
Example is using it for Integer.
https://stackoverflow.com/questions/28973229/javafx-binding-stringproperty-and-integerproperty
The idea is if the observable float field changes. the text should change.

Reply:
As we improve let's update the Confluence page detailing these behaviors for these controls. If/when this is understood we should apply to Title and Tooltip text.
This is important when we begin the effort when the user changes coordinates (filters). But also when we write a transaction the UI elements need to change.

Reply:
Ooops, I missed this comments but we talked about this in our meeting afterwards.